saleor/dashboard/staff/forms.py

```diff
@@ -0,0 +1,13 @@
+from django import forms
+from django.contrib.admin import widgets
+
+from ...userprofile.models import User
+
+
+class UserGroupForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ['groups']
+        widgets = {
+            "groups": widgets.FilteredSelectMultiple(('groups'), False)
```

Comment:
Why this widget? What's the reason to use it? It comes from `contrib.admin` which we rather don't use.

Reply:
Is more compact. With CheckboxSelectMultiple if we have many groups we are loosing a lot of space.

Reply:
Deleted. Standard Many2Many widget looks the same